packages/yew-functional-macro/tests/function_attr/generic-pass.rs

```diff
@@ -0,0 +1,40 @@
+#[derive(Clone, ::yew::Properties, PartialEq)]
+struct Props {
+    a: usize,
+}
+
+#[::yew_functional::function_component(Comp)]
+fn comp<P>(_props: &P) -> ::yew::Html
+where
+    P: ::yew::Properties + PartialEq,
+{
+    ::yew::html! {
+        <p></p>
+    }
+}
+
+#[::yew_functional::function_component(Comp1)]
+fn comp1<T1, T2>(_props: &()) -> ::yew::Html {
+    ::yew::html! {
+        <p></p>
+    }
+}
+
+// TODO: uncomment when min_const_generics are in stable and Rust version in CI is bumped
+// #[::yew_functional::function_component(ConstGenerics)]
+// fn const_generics<const N: i32>() -> ::yew::Html {
+//     ::yew::html! {
+//         <div>
+//             { N }
+//         </div>
+//     }
+// }
```

Comment:
Can't we just use a `#[cfg]` attribute checking for `const_generics` here?
I'm not a fan of including comments so it's either that or we remove it entirely.

Reply:
How would one introduce conditional compilation based on whether rust feature (`min_const_generics`)?

Reply:
Okay never mind, that doesn't really make sense because there's nothing to activate the feature in the first place :D
Let's just leave this for now because min_const_generics is gonna be released in the next stable version anyway IIRC.
Could you please open a new issue for this though so we don't forget?